test/ERC721Drop.t.sol

```diff
@@ -360,30 +359,55 @@ contract ERC721DropTest is Test {
         calls[0] = abi.encodeWithSelector(
             IERC721Drop.adminMint.selector,
             DEFAULT_OWNER_ADDRESS,
-            5 
+            5
         );
         calls[1] = abi.encodeWithSelector(
             IERC721Drop.adminMint.selector,
             address(0x123),
             3
         );
-        calls[2] = abi.encodeWithSelector(
-            IERC721Drop.saleDetails.selector
-        );
+        calls[2] = abi.encodeWithSelector(IERC721Drop.saleDetails.selector);
         bytes[] memory results = zoraNFTBase.multicall(calls);
 
-        (bool saleActive, bool presaleActive, uint256 publicSalePrice, , , , , , , ,) = abi.decode(results[2], (bool, bool, uint256, uint64, uint64, uint64, uint64, bytes32, uint256, uint256, uint256));
+        (
+            bool saleActive,
```

Comment:
nit: can we change our linter to keep these single line?

Reply:
Oof this is annoying. Can check.

Reply:
Seems to be more trouble than it's worth – i'll revisit this soon.